docs/2.guide/2.directory-structure/1.components.md

```diff
@@ -34,8 +34,16 @@ By default, only the `~/components` directory is scanned. If you want to add oth
 ```ts [nuxt.config.ts]
 export default defineNuxtConfig({
   components: [
-    { path: '~/components/special-components', prefix: 'Special' },
+    // ~/components/Btn.vue => <Btn />
+    // ~/components/base/Btn.vue => <BaseBtn />
     '~/components'
+
+    // ~/components/base/Btn.vue => <Btn />
+    { path: '~/components/base/', pathPrefix: false },
```

Comment:
this doesn't really make any difference in the example you use as there is no path refix. Maybe `~/components/base/further/Btn.vue` -> `<Btn />` would be a better example?